src/components/teamCard1_2023/TeamCard1_2023.css

```diff
@@ -0,0 +1,133 @@
+.codeutsava__team-card1_2023 {
+    padding: 50px 20px;
+    background-color: #fff;
+    text-align: center;
+    overflow: hidden;
+    position: relative;
+    border-radius: 20px;
+    min-width: 320px;
+    max-width: 320px;
+  }
+  
+  .codeutsava__team-card1_2023 .codeutsava__team-card1_2023-img{
+    display: inline-block;
+    height: 170px;
+    width: 170px;
+    margin-bottom: 15px;
+    z-index: 1;
+    position: relative;
+  }
+  
+  .codeutsava__team-card1_2023 .codeutsava__team-card1_2023-img::before {
+    content:"";
+    background-image: url(../../assets/images/teamcard1_2023/teamcard_4_web.png);
+  background-position: 0 50px;
+    background-size:cover;
+  
+    width: 100%;
+    height: 0;
+    border-bottom-left-radius: 85%;
+    border-bottom-right-radius: 20%;
+    background-color:purple;
+    position: absolute;
+    bottom: 135%;
+    right: 0;
+    left: 0;
+    opacity: 0.9;
+    transform: scale(3);
+    transition: all 0.4s linear 0s;
+  }
+  
+  .codeutsava__team-card1_2023:hover .codeutsava__team-card1_2023-img::before {
+    height: 100%;
+  }
+  
+  .codeutsava__team-card1_2023 .codeutsava__team-card1_2023-img::after {
+    content: "";
+    width: 100%;
+    height: 100%;
+    border-radius: 50%;
+   
+    background-color:purple;
+    position: absolute;
+    top: 0;
+    left: 0;
+    z-index: -1;
+  }
+  
+  .codeutsava__team-card1_2023 .codeutsava__team-card1_2023-img img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    border-radius: 50%;
+    transform: scale(1);
+    transition: all 0.6s ease 0s;
+  }
+  
+  .codeutsava__team-card1_2023:hover .codeutsava__team-card1_2023-img img {
+    box-shadow: 0 0 0 14px #fff;
+    transform: scale(0.75);
+  }
+  
+  .codeutsava__team-card1_2023 .name {
+    display: block;
+    font-size: 22px;
+    color: #000000;
+    text-transform: capitalize;
+  }
+  
+  .codeutsava__team-card1_2023 .title {
+    display: block;
+    font-size: 17px;
+    color:purple;
+    text-transform: capitalize;
+    margin-top: 4px;
+  }
+  
+  .domain3{
+    color: #000;
+    font-size: 0.8rem;
+    margin-bottom: 1rem;
+  }
+  
+  .codeutsava__team-card1_2023 .social {
+    width: 100%;
+    height: 50px;
+    padding: 0;
+    margin: 0;
+    background-color:purple;
+    position: absolute;
+    bottom: -100px;
+    left: 0;
+    transition: all 0.5s ease 0s;
+    background-image: url(../../assets/images/teamcard1_2023/teamcard_4_pumpkin.png);
```

Comment:
Remove background image of pumpkin here, and also remove background classes.
Only use ,
border-top-right-radius: 85%;
border-top-left-radius: 85%;
the above two css here for both side radius.